webpack.config.babel.js

```diff
@@ -155,25 +158,24 @@ module.exports = ({ stats = false } = {}) => ({
     new webpack.EnvironmentPlugin({
       NODE_ENV: 'development',
       BASE_URL: baseUrl,
+      // MOCK: Use mock or real API implementation
+      MOCK: 'false',
+      MOCK_WALLET: process.env.MOCK || 'false',
+      MOCK_TOKEN_LIST: process.env.MOCK || 'false',
+      MOCK_ERC20: process.env.MOCK || 'false',
+      MOCK_DEPOSIT: process.env.MOCK || 'false',
+      MOCK_EXCHANGE: process.env.MOCK || 'false',
+      MOCK_WEB3: process.env.MOCK || 'false',
+      // AUTOCONNECT: only applies for mock implementation
+      AUTOCONNECT: 'true',
     }),
     new ForkTsCheckerWebpackPlugin({ silent: stats }),
     // define inside one plugin instance
     new webpack.DefinePlugin({
       VERSION: JSON.stringify(require('./package.json').version),
       DEX_JS_VERSION: JSON.stringify(require('@gnosis.pm/dex-js/package.json').version),
       CONTRACT_VERSION: JSON.stringify(require('@gnosis.pm/dex-contracts/package.json').version),
-
-      // MOCK: Use mock or real API implementation
-      'process.env.MOCK': JSON.stringify(process.env.MOCK || 'false'),
-      'process.env.MOCK_WALLET': JSON.stringify(process.env.MOCK_WALLET || process.env.MOCK || 'false'),
-      'process.env.MOCK_TOKEN_LIST': JSON.stringify(process.env.MOCK_TOKEN_LIST || process.env.MOCK || 'false'),
-      'process.env.MOCK_ERC20': JSON.stringify(process.env.MOCK_ERC20 || process.env.MOCK || 'false'),
-      'process.env.MOCK_DEPOSIT': JSON.stringify(process.env.MOCK_MOCK_DEPOSIT || process.env.MOCK || 'false'),
-      'process.env.MOCK_EXCHANGE': JSON.stringify(process.env.MOCK_EXCHANGE || process.env.MOCK || 'false'),
-      'process.env.MOCK_WEB3': JSON.stringify(process.env.MOCK_WEB3 || process.env.MOCK || 'false'),
-
-      // AUTOCONNECT: only applies for mock implementation
-      'process.env.AUTOCONNECT': JSON.stringify(process.env.AUTOCONNECT || 'true'),
+      CONFIG: JSON.stringify(config),
```

Comment:
I still think this is a bad idea on account of no treeshaking
I still think a good loader would be better
I'll need experiment with that

Reply:
You mean putting all the config into a global var?
In terms of the effects on treeshaking, how big does the config have to be in order to have an actual impact? I have no idea.

Reply:
I'll test stuff tomorrow. Will see

Reply:
:crossed_fingers: